articles/tales-from-fleet-security-securing-bank-accounts-from-business-email-compromise.md

```diff
@@ -0,0 +1,112 @@
+
+According to the [FBI](https://www.ic3.gov/Media/Y2022/PSA220504), business email compromise (BEC) attacks are increasing in popularity yearly, costing organizations over 43 billion dollars in losses between 2016 and 2021.
+
+As BEC is a form of social engineering, we often think education is the only way to reduce the odds of getting compromised. 
+
+Security teams often focus on protecting IT assets only, but what could be more damaging to your company than having its bank account emptied fraudulently?
+
+Since BEC typically targets one straightforward thing, money, there are controls we can implement to reduce the risk significantly. The parameters suggested in this post are those we found the most useful through our research. Unlike previous posts in this series, we cannot be as specific about how to enable them as every bank is different.
+
+## Email configuration
+While good email hygiene can't stop BEC, it can at least reduce the number of fraudulent emails that make it through and force criminals to work harder.
+
+1. Require secure two-factor authentication (2FA) to access email, such as [FIDO U2F](https://fidoalliance.org/specs/u2f-specs-master/fido-u2f-overview.html) security keys. Using U2F keys makes the credentials phishing-resistant, lowering the risk of malicious access to your email accounts from being used to launch an internal BEC attempt.
+2. Configure email security features such as [SPF, DMARC, and DKIM](https://support.google.com/a/answer/2466580?product_name=UnuFlow&hl=en&visit_id=637919491839685136-2075570827&rd=1&src=supportwidget0&hl=en). These help make it harder to impersonate your domain name in an email.
+3. Configure spam filters to protect your organization from people impersonating employees from other domain names. On Google Workspace/Gmail, this setting is called **Protect against spoofing of employee names**, and you should enable it along with **Protect against inbound emails spoofing your domain**. 
+
+You can find detailed instructions on configuring Google Workspace in our [handbook](https://fleetdm.com/handbook/security#email-authentication).
+
+## Online banking configuration
+
+The best email configuration ever is not enough to protect from BEC and other types of social engineering attacks. Something will always slip by, and we must assume that even well-trained employees will fall for a good enough attempt. 
+
+If you think you can't fall victim to these emails, you probably have not received one that was sufficiently well-targeted and delivered at precisely the right time.
+
+As defenders, we often say that while we must always be perfect, attackers only need to succeed once. As [Joe Slowik](https://www.dragos.com/blog/industry-news/the-myth-of-the-adversary-advantage/) explained in this excellent blog post a couple of years ago, that is seldom true. As defenders, we have the high ground, but we often fail to leverage it.
```

Comment:
```suggestion
As defenders, we often say that while we must always be perfect, attackers only need to succeed once. As [Joe Slowik](https://www.dragos.com/blog/industry-news/the-myth-of-the-adversary-advantage/) explained in this excellent blog post a couple of years ago, that is seldom true. We defenders have the high ground, but we often fail to leverage it.
```

Repetition. I suggest rephrasing to cut out the repeated phrase "as defenders" and avoid starting three sentences in a row with "As."